gaj23/rancid
# Rlt/feature/router error

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- `React Router` implemented
- `Cypress` tests began 
- fixes some error handling
- moves api calls to seperate file 


## *Instructions for reviewer:*
*Where should the reviewer start?*
- open the page and check for errors in regular use

*How should this be manually tested?*
1. `Router`
- open the page and click on a movie card
- observe the url including the movie's id

2. `Cypress`
- run `npm run cypress` on the command line
- cypress should open and run tests
- the tests should pass

3. Error-handling
- change url in `src/App.js` at line 20 to `https://rancid-tomatillos.herokuapp.com/api/v2/moves/`
   - `Error` component should display
- change the same url to `https://rancid-tomatillos.herouapp.com/api/v2/movies/`
   - `Error` component should display
- change url in `src/Film.js` to `https://rancid-tomatillos.herokuapp.com/api/v2/movies/1`
   - `Error` component should display

*Any background context you want to provide?*
*What are the relevant tickets?*
- closes #6 
- closes #18 
- closes #19 
- closes #11 
- closes #22 
- closes #23 


## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*



## Checklist:
